scenes/quests/lore_quests/quest_002/4_closing_transition/components/closing_cinematic.dialogue

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: The Threadbare Authors
+# SPDX-License-Identifier: MPL-2.0
+~ start
+do animation_player.play("enter")
+do animation_player.animation_finished
+Oh what do you have there? Is that a story from the monk?
+Tell us! Tell us!
+do Transitions.do_transition(reveal)
```

Comment:
This causes the music to fade out and not fade back in. I wonder why! But I think we can address that later.